src/spryker/application/skeleton/src/Inviqa/Yves/EventDispatcher/EventDispatcherDependencyProvider.php

```diff
@@ -17,7 +17,7 @@ class EventDispatcherDependencyProvider extends PyzEventDispatcherDependencyProv
     /**
      * @override
      *
-     * @return \Spryker\Shared\EventDispatcherExtension\Dependency\Plugin\EventDispatcherPluginInterface[]
+     * @return array<\Spryker\Shared\EventDispatcherExtension\Dependency\Plugin\EventDispatcherPluginInterface>
```

Comment:
:( I hope coding standards doesn't force this now

Reply:
although technically it should be the namespace `use` as well

Reply:
That was a modification due to code standards complaining, yes.